### PR TITLE
Carthage content benchmarking: Update from carthage

### PR DESCRIPTION
**Pull request by Lezek123**
Commits of interest:

* Update after 1st merge: https://github.com/Joystream/joystream/pull/4221/commits/91c5704978ce4542d934c01f6e95ade531d7ac8a
* Update after 2nd merge: https://github.com/Joystream/joystream/pull/4221/commits/017ccf79885d7d31fe81b4bb2244876833f6d27f
* Clippy fixes: https://github.com/Joystream/joystream/pull/4221/commits/04ef650a91ef339b79e93de36ef41ff0abeeb43f
* Addressing https://github.com/Joystream/joystream/issues/4262: https://github.com/Joystream/joystream/pull/4221/commits/6229c725a9e2e2a01c2109f7232d459a36769981

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1202132419573087/1202942756873123) by [Unito](https://www.unito.io)

### CODE REVIEW

File: runtime-modules/content/src/benchmarks/benchmarking.rs [hunk 1]
```diff
@@ -36,12 +36,12 @@ use super::{
     assert_last_event, channel_bag_witness, clone_curator_group,
     create_data_object_candidates_helper, generate_channel_creation_params, insert_content_leader,
     insert_curator, insert_distribution_leader, insert_storage_leader, max_curators_per_group,
-    member_funded_account, setup_video, setup_worst_case_curator_group_with_curators,
```
Comment (dobertRowneySr) [inline runtime-modules/content/src/benchmarks/benchmarking.rs:39]: why not simply the `use *` version?

**Review by dobertRowneySr — COMMENTED** [review]
LGTM,
the benchmarks are updated and with the new utility functions.
CI checks are ok too except for the joystream types one which is failing.

File: runtime-modules/content/src/benchmarks/benchmarking.rs (hunk 1 above)
Comment (Lezek123) [reply runtime-modules/content/src/benchmarks/benchmarking.rs:39]: Addressed in https://github.com/Joystream/joystream/pull/4221/commits/c625d8a09b40545ffb5d8259c75d8571249545b8

**Review by mnaamani — APPROVED** [review]
Tested content benchmark tests, and weight generation.
Looks like carthage  has advanced a bit since this update but otherwise looks good.

**Pull request merged**